make.sh

```diff
@@ -80,7 +80,7 @@ rule() {
     ;; dev)
       eval $(opam env)
       echo "Installing opam packages for development..."
-      opam install -y utop ocaml-lsp-server ocp-indent ocamlformat ounit2 earlybird
+      opam install -y utop ocaml-lsp-server ocp-indent ocamlformat ounit2
```

Comment:
The debugging documentation refers specifically to `make dev` for installing earlybird. That should be updated as well then.